### DISCUSSION TITLE
SendGrid Onboarding Experience

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1328] [Office hours] [answered]
I have a question regarding the use of Disclosure component.

I'm working on making updates to the SendGrid onboarding experience which is the attached screenshot. The "Create a sender identity" and "Send your first email" are using an accordion from the SendGrid legacy design system. I was curious if other people have grouped disclosures to make a similar accordion effect. Open to other ways that we can achieve the same thing without having to try and replicate an accordion.

Some other areas I could use some input on are the background gradient and illustrations being used.

Thanks,
Wilfredo
![Screen Shot 2021-03-29 at 3 39 33 PM](https://user-images.githubusercontent.com/81593493/112908973-b841e400-90ad-11eb-8b5e-729d453c6121.png)

**Comment by serifluous** [comment, marked as answer]
Thanks for sharing, @wilfredomerced ! We'll take a look before office hours next week.

**Reply by gloriliale** [reply]
Video of the discussion during Office Hours here (starts at 1:42)- https://www.loom.com/share/45266766429f4455b99e94e6c7af1cca
